src/test/java/teammates/test/cases/browsertests/InstructorHelpPageUiTest.java

```diff
@@ -0,0 +1,82 @@
+package teammates.test.cases.browsertests;
+
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import teammates.common.util.Const;
+import teammates.test.pageobjects.AppPage;
+import teammates.test.pageobjects.InstructorHelpPage;
+
+/**
+ * SUT: {@link InstructorHelpPage}.
+ */
+public class InstructorHelpPageUiTest extends BaseUiTestCase {
+    private InstructorHelpPage helpPage;
+
+    @Override
+    protected void prepareTestData() {
+        // no test data used in this test
+    }
+
+    @BeforeClass
+    public void classSetup() {
+        helpPage = AppPage.getNewPageInstance(browser, createUrl(Const.ActionURIs.INSTRUCTOR_HELP_PAGE + ".jsp"),
+                                                InstructorHelpPage.class);
+    }
+
+    @Test
+    public void testAll() throws Exception {
+        testSearch();
+    }
+
+    private void testSearch() throws Exception {
+
+        ______TS("content: default help page");
+
+        helpPage.verifyHtmlMainContent("/instructorHelpPageDefault.html");
+
+        ______TS("search for empty keyword");
+
+        helpPage.clearSearchBox();
+        helpPage.clickSearchButton();
+        helpPage.verifyHtmlMainContent("/instructorHelpPageDefault.html");
+
+        ______TS("search for single keyword");
+
+        String searchContent = "course";
+        helpPage.inputSearchQuery(searchContent);
+        helpPage.clickSearchButton();
+        helpPage.verifyHtmlMainContent("/instructorHelpPageSearchSingleKeyword.html");
+
+        ______TS("search for multiple keywords");
+
+        searchContent = "enroll students";
+        helpPage.clearSearchBox();
+        helpPage.inputSearchQuery(searchContent);
+        helpPage.clickSearchButton();
+        String searchResultsMultipleKeywords = helpPage.getSearchResults();
+        helpPage.verifyHtmlMainContent("/instructorHelpPageSearchMultipleKeywords.html");
+
+        ______TS("check case insensitivity of query");
+
+        searchContent = "eNroLL STUDENTs";
+        helpPage.clearSearchBox();
+        helpPage.inputSearchQuery(searchContent);
+        helpPage.clickSearchButton();
+        assertEquals(searchResultsMultipleKeywords, helpPage.getSearchResults());
```

Comment:
I was just thinking is a file needed now since I am just comparing the search meta data which is a sentence. 
If I compare it against a file I will need to lowercase the content first since we need to ignore the case ?

Reply:
The file only contain part of the page rather than the whole page.

Reply:
Ok makes sense. However, another question I had was how would I reuse the file for case insensitive  test ? Essentially I'm trying to check that `43 results found for enroll students` is same as `43 results found for ENROLL STUDENTS` but ignoring the case of the query. I direct comparison against the file wouldn't work right.

Reply:
Good questions... I thought your result DIV doesn't contain the information but pure results. NVM, you can keep the current way and prepare your staging server.

Reply:
Apologies I think I'm still not being very clear :sweat_smile: 
Basically there are 2 divs `searchMetaData` and `searchResults`. 

`searchMetaData` describes the number of results returned like i mentioned above (`43 results found for enroll students`)
`searchResults` is a list of pure result panels.

When you said we should check that no. of results returned is same I thought you meant `searchMetaData` but I think you actually mean the latter. In that case what you're saying will work since `searchResults` contains pure results. 

Sorry for the confusion. I'll make the changes